src/main/java/org/wso2/siddhi/plugins/idea/formatter/SiddhiFormattingModelBuilder.java

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.siddhi.plugins.idea.formatter;
+
+import com.intellij.formatting.FormattingModel;
+import com.intellij.formatting.FormattingModelBuilder;
+import com.intellij.formatting.FormattingModelProvider;
+import com.intellij.formatting.Indent;
+import com.intellij.formatting.SpacingBuilder;
+import com.intellij.lang.ASTNode;
+import com.intellij.openapi.util.TextRange;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.codeStyle.CodeStyleSettings;
+import org.jetbrains.annotations.Nullable;
+import org.wso2.siddhi.plugins.idea.SiddhiLanguage;
+
+import javax.annotation.Nonnull;
```

Comment:
Why is this used instead of `org.jetbrains.annotations.NotNull`?

Reply:
FindBugs doesn't allow to use org.jetbrains.annotations.NotNull instead of javax.annotation.Nonnull.

Reply:
Fixed in commit 3aa87a6